macros/blueprint-proc-macro/src/job.rs

```diff
@@ -17,6 +17,7 @@ mod kw {
     syn::custom_keyword!(verifier);
     syn::custom_keyword!(evm);
     syn::custom_keyword!(event_handler);
+    syn::custom_keyword!(event_listener);
     syn::custom_keyword!(protocol);
```

Comment:
Can you also remove this and remove the need for this? It seems superfluous given the introduction of this abstraction. Lmk if you think otherwise for now. I prefer the 2 birds w/ 1 stone approach whenever possible.

Reply:
This is a good refactoring task. After much friction, I decided to shelve my changes and move forward since what we have now works, and will adds improvements to the job macro with respect to ordering params.